pkgs/development/libraries/gstreamer/core/default.nix

```diff
@@ -108,6 +109,15 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.mesonEnable "libdw" (withLibunwind && hasElfutils))
   ];
 
+  patches = [
+    (fetchpatch {
+      name = "c-args.patch";
+      url = "https://gitlab.freedesktop.org/gstreamer/gstreamer/-/commit/9c0fbfd2eb3ae7a88099db86b7e21c2e7e5c8909.patch";
+      hash = "sha256-miHJ1Iq1Q0xmClf5laTRNwBTqGq4bgT1YTZLdPejq4E=";
+    })
+  ];
+  patchFlags = "-p3";
```

Comment:
It is better to do this transformation in `fetchpatch` so that we do not need to apply it to all patches.

Reply:
I am fairly sure -p3 is a desired flag since it is required to apply any patch meant for the upstream source repository.